tests/ir/tsl/test_tiled_stride.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+
+from compiler.ir.tsl.stride import Stride
+from compiler.ir.tsl.tiled_stride import TiledStride
+
+
+@pytest.fixture()
+def example_strides():
+    return (Stride(1, 4), Stride(4, 6), Stride(24, 2), Stride(None, None))
+
+
+@pytest.fixture()
+def example_tiled_strides(example_strides):
+    stride1, stride2, stride3, dynamic_stride = example_strides
+    tiledStride1 = TiledStride([stride1, stride2])
+    tiledStride2 = TiledStride([stride1, stride2, stride3])
+    tiledStride3 = TiledStride([stride1, dynamic_stride])
```

Comment:
I find it a bit confusing that the constructor is written like this:
```
Stride(1, 4), Stride(None, None)
```
Yet the layout is written like:
```
"[1, ?] * [4, ?]"
```

Reply:
Hmm yes but is this really a problem?